docs/operating-scylla/procedures/cluster-management/replace-dead-node.rst

```diff
@@ -189,11 +189,11 @@ In this case, the node's data will be cleaned after restart. To remedy this, you
 
       sudo sed -e '/auto_bootstrap:.*/s/False/True/g' -i /etc/scylla/scylla.yaml
 
-#. Run the following command, replacing 172.30.0.186 with the listen_address / rpc_address of the node that you are restarting:
+#. Run the following command to replace the instance whose ephemeral volumes were erased (previously known by the Host ID of the node you are restarting) with the restarted instance. The restarted node will be assigned a new random Host ID.
```

Comment:
Same comment re instance vs host vs node.

Reply:
Here instance refers to a cloud VM that lost its ephemeral storage, so I think it does make sense in this context, but it can probably be said more clearly.  How about:

```
To restore the data lost by the restarted instance, locate the Host ID of the respective Scylla node and use the following option when restarting the node to initiate its replacement. The restarted node will be assigned a new random Host ID.
```

Reply:
Maybe I am mistaken, but I think we're not restoring any data when replacing? We're restoring the replication factor, but the data is streamed from other nodes, it's not taken from some volume locally? Now I don't get how can you *restart* a node yet change its startup options, given its instance (VM) was lost? Rather, we're starting a fresh new node with intention to replace the node lost along with the permanently lost VM. Am I getting this right?

Reply:
It comes down to the terminology, that I don't think our documentation is crispy clear about.
This scenario calls for a cloud instance that loses the data volume.  When restarted, scylla will pick a new random host_id and the replace option (to scylla, not to the instance) tells it to replace the dead node that was lost, that happened to be served on the same instance, but it's almost irrelevant (except maybe for reusing the same ip address).
The node's content is rebuilt from the remaining replicas.

Reply:
In that case I think it would be more clear to say that we're replacing the node and stream its data set from the remaining replicas. The only thing that stays the same is IP address and tokens.

Reply:
Rephrased in [07d8629](https://github.com/scylladb/scylladb/commit/07d8629e2bd922483cb2000d26f5bf285def0b9a)